src/commands/ai/sdxl_turbo.ts

```diff
@@ -0,0 +1,62 @@
+import type { Command } from '@commands';
+import { SlashCommandBuilder } from 'discord.js';
+
+import { EventReader } from './hugging_face.ts';
+
+const DEFAULT_VALUE = {
+    strength: 0.7,
+    steps: 2,
+    get seed(): number {
+        return Math.floor(Math.random() * 12013012031030);
+    },
+};
+
+/*
+ * @command     - sdxl_turbo
+ * @description - Génère des images avec SDLXL Turbo!
+ * @permission  - None
+ */
+export const SDXL_TURBO: Command = {
+    data: new SlashCommandBuilder()
+        .setName('sdxl_turbo')
+        .setDescription('Génère des images avec SDLXL Turbo!')
+        .addStringOption((option) => option.setName('prompt').setDescription('Le prompt').setRequired(true))
+        .addNumberOption((option) =>
+            option.setName('strength').setDescription('La force du bruitage (0.7 par défaut)').setRequired(false),
+        )
+        .addNumberOption((option) =>
+            option.setName('steps').setDescription("Le nombre d'étapes (2 par défaut)").setRequired(false),
+        )
+        .addNumberOption((option) =>
+            option.setName('seed').setDescription('La graine (aléatoire par défaut)').setRequired(false),
+        ),
+    async execute(interaction) {
+        await interaction.deferReply();
+        const replyError = async (msgError: string): Promise<void> => {
+            await interaction.editReply(msgError);
+        };
```

Comment:
global function/utility class ?

Reply:
*magic number* from the api. made a const on the top.

Reply:
`interaction` is a captured variable

Reply:
Either pass the interaction or yeah throw (will need to show the right error message in the interactionCreate event